src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -149,13 +245,20 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     void ColorSchemes::ColorPickerChanged(IInspectable const& sender,
                                           ColorChangedEventArgs const& args)
     {
-        if (auto picker = sender.try_as<ColorPicker>())
+        if (const auto picker{ sender.try_as<ColorPicker>() })
         {
-            if (auto tag = picker.Tag())
+            if (const auto tag{ picker.Tag() })
             {
-                auto index = winrt::unbox_value<uint8_t>(tag);
+                const auto index{ winrt::unbox_value<uint8_t>(tag) };
                 CurrentColorScheme().SetColorTableEntry(index, args.NewColor());
-                _CurrentColorTable.GetAt(index).Color(args.NewColor());
+                if (index < ColorTableDivider)
+                {
+                    _CurrentNonBrightColorTable.GetAt(index).Color(args.NewColor());
+                }
+                else
+                {
+                    _CurrentBrightColorTable.GetAt(index).Color(args.NewColor());
+                }
```

Comment:
_what_, now THIS seems broken in the way @htcfreek was talking about because it does not have the -8

Reply:
or specifically -Divider